packages/react/src/categories/utils/getRootCategory.ts

```diff
@@ -0,0 +1,14 @@
+import type { Category } from '@farfetch/blackout-client';
```

Comment:
I think this module should be at least inside the `@farfetch/blackout-redux` because if the developer does not want to use the hook and opts to use the redux's actions and selectors, he shouldn't have to add the `@farfetch/blackout-react` dependency for that. Since the `@farfetch/blackout-redux` is a dependency of `@farfetch/blackout-react` both of those use cases benefit from that change. What do you think?

Reply:
Makes sense!!👍